app/managers.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema")
+        self.table_name = "actors"
+
+    def create(self, first_name_: str, last_name_: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} "

```

Comment:
It's not critical in this case, but I'd still suggest to format raw SQL according to SQL style guides. e.g.
```python
        self._connection.execute(
            f"INSERT INTO {self.table_name} (first_name, last_name) "
            "VALUES (?, ?)",
            (first_name_, last_name_)
        )
```
Same at line $21

Reply:
I did this to avoid conflicts with column names in the table

Reply:
It's about "_"

Reply:
Wow. But there can be no conflicts there. Those are not only in different namespaces, they are in different programs.

Reply:
Okay, I see)